common_benchmark_suite/openxla/benchmark/unique_ids.py

```diff
@@ -56,6 +56,34 @@
 OUTPUT_DATA_BERT_LARGE_FP16_JAX_384X1024XF16 = f"{OUTPUT_DATA_BERT_LARGE_JAX_384X1024}-fp16"
 OUTPUT_DATA_BERT_LARGE_BF16_JAX_384X1024XBF16 = f"{OUTPUT_DATA_BERT_LARGE_JAX_384X1024}-bf16"
 
+################################################################################
+# ResNet50 models                                                              #
+################################################################################
+MODEL_IMPL_RESNET50 = f"aff75509-4420-40a8-844e-dbfc48494fe6-MODEL_IMPL"
+MODEL_RESNET50 = "aff75509-4420-40a8-844e-dbfc48494fe6-MODEL_RESNET50"
```

Comment:
Might need a `MODEL_RESNET` and then a `MODEL_RESNET50`, as `MODEL_BERT` is to `MODEL_BERT_LARGE`.

Reply:
Done.